salt/top.sls

```diff
@@ -21,12 +21,13 @@ base:
     - flannel
     - docker
     - container-feeder
+    - kubectl-config
     - kubelet
     - kube-proxy
   'roles:kube-master':
     - match: grain
     - kube-apiserver
     - kube-controller-manager
     - kube-scheduler
-    - kubectl-client-cert
     - addons
+    - dex
```

Comment:
I think I would put dex before the addons, as addons might rely on RBAC stuff being up-to-date

Reply:
No, this isn't important. Creation of the addons all use the kubectl config, which is cert-based auth, not username/password.